src/screens/LoginScreen.js

```diff
@@ -1,52 +1,69 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
 import {
   SafeAreaView,
   StyleSheet,
   View,
   Text,
   Image,
   StatusBar,
+  Button,
 } from 'react-native';
 import * as Linking from 'expo-linking';
 
 import Login from '../components/Login/Login';
 import s4sLogo from '../../assets/images/s4s-logo.png';
 import Colors from '../constants/Colors';
+import mockBundle from '../../assets/mock_data/bundle-blake-eichmann.json';
+import { setPatientData } from '../features/patient/patientDataSlice';
 
-const LoginScreen = ({ navigation }) => (
-  <SafeAreaView style={styles.safeAreaView}>
-    <StatusBar backgroundColor={Colors.primary} barStyle="dark-content" />
-    <View
-      contentInsetAdjustmentBehavior="automatic"
-      style={styles.screen}
-    >
-      <View style={styles.discoveryContainer}>
-        <View style={styles.logoContainer}>
-          <Image
-            style={styles.slogo}
-            source={s4sLogo}
-            resizeMode="contain"
-          />
-          <View style={styles.descriptionContainer}>
-            <Text style={styles.descriptionText}>Discovery Mobile</Text>
+const LoginScreen = ({ navigation, setPatientDataAction }) => {
+  const handleSkipLogin = () => {
+    setPatientDataAction(mockBundle);
+    navigation.navigate('PostAuth');
+  };
+
+  return (
+    <SafeAreaView style={styles.safeAreaView}>
+      <StatusBar backgroundColor={Colors.primary} barStyle="dark-content" />
+      <View
+        contentInsetAdjustmentBehavior="automatic"
+        style={styles.screen}
+      >
+        <View style={styles.discoveryContainer}>
+          <View style={styles.logoContainer}>
+            <Image
+              style={styles.slogo}
+              source={s4sLogo}
+              resizeMode="contain"
+            />
+            <View style={styles.descriptionContainer}>
+              <Text style={styles.descriptionText}>Discovery Mobile</Text>
+            </View>
           </View>
+          <Login navigation={navigation} />
+        </View>
+        <View style={styles.vermonsterContainer}>
+          <Button title="Skip Login" onPress={handleSkipLogin} />
```

Comment:
Can this be disabled based on some ENV?   (perhaps development mode -- if deploys are _not_ development mode?)

Reply:
+1

Added a `process.env` lookup for development